pandas/tests/io/formats/test_info.py

```diff
@@ -82,18 +55,48 @@ def test_info_categorical_column():
     df2.info(buf=buf)
 
 
-def test_info(float_frame, datetime_frame):
-    io = StringIO()
-    float_frame.info(buf=io)
-    datetime_frame.info(buf=io)
+@pytest.mark.parametrize(
```

Comment:
i think can you do this if you don't list the reference to the fixture and instead just use a string, the `.getfixturevalue` takes a string any how, see: https://stackoverflow.com/questions/42014484/pytest-using-fixtures-as-arguments-in-parametrize

Reply:
@jreback, thank you very much! It worked.